docs/tutorial/rewriter/examples/outputs_option.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""ONNX Pattern Rewriting with output specification
+
+This script shows how to define a rewriting rule that specifies
+the number and names of outputs from operations.
+"""
+
+import onnx
+
+import onnxscript
+from onnxscript import FLOAT, opset18, script
+from onnxscript.rewriter import pattern
+
+
+@script()
+def original_model(A: FLOAT[4, 4]) -> FLOAT[2, 4]:
+    # Split operation that produces 2 outputs
+    result1, result2 = opset18.Split(A, num_outputs=2, axis=0)
+    # We only return the first output for simplicity
+    return result1
+
+
+_model = original_model.to_model_proto()
+onnx.checker.check_model(_model)
+
+
+####################################
+# The target pattern with multiple outputs
+# =====================
+
+
+def split_pattern(op, input):
+    # Pattern to match Split operations with 2 outputs
+    # _outputs=2 specifies that this operation produces 2 outputs
+    return op.Split(input, num_outputs=2, axis=0, _outputs=2)
+
+
+####################################
+# The replacement pattern with named outputs
+# =====================
+
+
+def custom_split_replacement(op, input, **_):
+    # Replace with a custom split operation using named outputs
+    # _outputs=["first_half", "second_half"] assigns names to the outputs
+    return op.CustomSplit(input, _domain="custom.domain", _outputs=["first_half", "second_half"])
+
+
+####################################
```

Comment:
Sorry, the single and triple replacements are not correct. Can you remove them from the example? It is important that the number of output values of the pattern (2 in the above split-pattern) match the number of output values of the replacement (as in custom_split_replacement). May be you can clarify this in the documentation?